### PR TITLE
Open.fm web site web-scrobbler add

### PR DESCRIPTION
**Pull request by JeonHyungJin**
Hi.
I added the open.fm web site`s scrobbler connector.
This PR is related with the issue #1823 .
Please check this PR and merge or comment about this PR.
Thanks and have a nice day :)

### CODE REVIEW

**Comment by ShinJii89** [comment]
How can I check this if I have only firefox addon? :D

**Comment by JeonHyungJin** [comment]
Hello! @ShinJii89  
Before this PR is being merged in this project, maybe you can check through [my repository](https://github.com/JeonHyungJin/web-scrobbler).

You can compile this project through this [following commands](https://github.com/web-scrobbler/web-scrobbler/wiki/Setup-development-environment).
such as

> git clone https://github.com/JeonHyungJin/web-scrobbler.git
> cd web-scrobbler
> npm install
> grunt compile:firefox
and follow [this methods](https://github.com/web-scrobbler/web-scrobbler/wiki/Install-a-temporary-add-on)

File: src/core/connectors.js [hunk 1]
```diff
@@ -733,6 +733,12 @@ define(function() {
 		js: ['connectors/bugs.js'],
 	},
 
+	{
+		label: 'openfm',
+		matches: ['*://open.fm*'],
```
Comment (inverse) [inline src/core/connectors.js]:
```suggestion
		matches: ['*://open.fm/*'],
```

**Comment by JeonHyungJin** [comment]
@inverse 
Ok! thanks for your advice.
I will check that part and then i will make another PR.

**Comment by inverse** [comment]
You don't have to close the PR - just create another commit :+1:

**Comment by JeonHyungJin** [comment]
oh! thanks for advice :)
I changed that part 👍

**Pull request merged**